backend/tests/test_modules.py

```diff
@@ -79,7 +79,7 @@ def test_cld_harris() -> None:
     OUTPUT_CSV_FILE = os.path.join(OUTPUT_DIR, OUTPUT_CSV_FILENAME)
     print(f"INPUT_FILE={os.path.abspath(INPUT_FILE)}")
     print(f"OUTPUT_CSV_FILE={os.path.abspath(OUTPUT_CSV_FILE)}")
-    cld_harris.execute(INPUT_FILE, OUTPUT_CSV_FILE)
+    cld_harris.execute(INPUT_FILE, OUTPUT_CSV_FILE, 27.1, 27.7)
```

Comment:
Surprised by constants hard written in a function call.
Good practice would be to declare it at the beginning of the file (or in a dedicated separate file) with a CAPITAL_NAME.

Reply:
Thank you for your remarks, indeed as it was tests and I was changing the parameters to test the algo, I hadn't bothered to do things properly, it's corrected:)